### PR TITLE
Added the `index_url` configuration

### PR DESCRIPTION
**Pull request by adriancuadrado**
This configuration is mentioned in [this article](https://docs.microsoft.com/en-us/cli/azure/azure-cli-extensions-overview#install-extensions-manually), but not in the table of this file I just edited.

### CODE REVIEW

**Comment by chasewilson** [comment]
This looks good to me. @jiasli, can you confirm for technical details?

File: docs-ref-conceptual/azure-cli-configuration.md [hunk 1]
```diff
@@ -110,6 +110,7 @@ When you provide a default value, that argument is no longer required by any com
 | __cloud__ | name | string | The default cloud for all `az` commands.  The possible values are  `AzureCloud` (default), `AzureChinaCloud`, `AzureUSGovernment`, `AzureGermanCloud`. To change clouds, you can use the `az cloud set –name` command.  For an example, see [Manage Clouds with the Azure CLI](manage-clouds-azure-cli.md). |
 | __extension__ | use_dynamic_install | string | Install an extension if it's not added yet when running a command from it. The possible values are `no` (default), `yes_prompt`, `yes_without_prompt`. |
 | | run_after_dynamic_install | boolean | Continue to run the command when an extension is dynamically installed for it. Default is `False`. |
+| | index_url | string | URL of private extension index file following the format in [index.json](https://github.com/Azure/azure-cli-extensions/blob/master/src/index.json). Once specified, executing `az extension add --name <extension-name>` will use that file to find the extension to add. |
```
Comment (kairu-ms) [inline docs-ref-conceptual/azure-cli-configuration.md:113]:
```suggestion
| | index_url | string | URL of private extension index file following the format in [index.json](https://github.com/Azure/azure-cli-extensions/blob/main/src/index.json). Once specified, executing `az extension add --name <extension-name>` will use that file to find the extension to add. |
```

**Comment by kairu-ms** [comment]
LGTM

**Comment by chasewilson** [comment]
Once we get the suggestion from @kairu-ms committed, we should be able to merge this. @adriancuadrado does his suggestion look good to you?

**Comment by adriancuadrado** [comment]
@chasewilson Yes. I just made the change right now. You can merge it.

**Pull request merged**